site/docs/v1/tech/samlv2/_saml_limitations.adoc

```diff
@@ -2,14 +2,18 @@ FusionAuth supports SAML both as an Identity Provider (IdP) and as a Service Pro
 
 The IdP implementation has certain limitations. Version `2.0` is supported; other SAML versions are unsupported.
 
-FusionAuth supports two `NameIDPolicy` values:
+Please examine the documented `NameIDPolicy` of the software package for which you are configuring FusionAuth to act as the IdP to determine the correct value.
+
+[NOTE.since]
+====
+Since version 1.28, FusionAuth supports any `NameIDPolicy` value.
```

Comment:
When we are the IdP, starting in 1.28.0 won't fail validation based upon NameId Policy. However we don't technically support any policy value. 

If the requested NameId policy is Persistent, we will use that, else we will use email. So technically we are supporting Email, Persistent and un-specified. 

```
urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress
urn:oasis:names:tc:SAML:2.0:nameid-format:persistent
urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified
```

Reply:
Okay, made changes that I hope clarify this.